main.cpp

```diff
@@ -21,13 +21,42 @@
 #endif
  
 #define MAX_THREAD_STATS    0x8
- 
+
+// Initialise the digital pin LED1 as an output
+DigitalOut led1(LED1);
+static EventFlags idle_ef;
+#define BLINKY_THREAD_STACK 224
```

Comment:
Could you move all the `#define` in the same location, i.e. after `MAX_THREADS_STATS`?